core/spec/models/spree/store_spec.rb

```diff
@@ -37,19 +37,32 @@
   end
 
   describe ".default" do
-    let!(:store)    { create(:store) }
-    let!(:store_2)  { create(:store, default: true) }
+    before do
+      Spree::Store.delete_all
```

Comment:
We need to be able to assume that we have a reasonable test framework and that the database is empty at the start of our tests. Otherwise we'd need to put `delete_all`s before practically every test we have.